apps/app/pages/lectures/[role]/[id].tsx

```diff
@@ -107,7 +107,9 @@ function Lectures() {
             </Row>
             <Row align="middle" gutter={[16, 16]}>
               <Col>
-                <Text>{`Ninja: ${lecture.ninja.first_name} ${lecture.ninja.last_name}`}</Text>
+                <Link href={`/profile/ninja/${lecture.ninja.id}`}>
+                  <Text>{`Ninja: ${lecture.ninja.first_name} ${lecture.ninja.last_name}`}</Text>
+                </Link>
```

Comment:
Wrap this around an `<a>` tag so when you hover over it, the mouse pointer changes